project/AppDependencies.scala

```diff
@@ -17,11 +17,11 @@ object AppDependencies {
   val test: Seq[ModuleID]    = Seq(
     "uk.gov.hmrc.mongo"     %% "hmrc-mongo-test-play-28"   % mongoHmrcVersion,
     "uk.gov.hmrc"           %% "bootstrap-test-play-28"    % bootstrapPlayVersion,
-    "org.scalatest"         %% "scalatest"                 % "3.2.16",
+    "org.scalatest"         %% "scalatest"                 % "3.2.17",
     "org.jsoup"              % "jsoup"                     % "1.16.1",
     "com.github.tomakehurst" % "wiremock-standalone"       % "2.27.2",
```

Comment:
Please update to `"org.wiremock" % "wiremock-standalone"  % "3.0.4"`, thanks. groupID has changed to [org.wiremock](https://mvnrepository.com/artifact/org.wiremock/wiremock-standalone/3.0.4).